source/guide_headplane.rst

```diff
@@ -0,0 +1,181 @@
+.. author:: Lukas Wolfsteiner <https://lukas.wolfsteiner.media>
+
+.. tag:: vpn
+.. tag:: wireguard
+.. tag:: self-hosting
+.. tag:: lang-nodejs
+.. tag:: web
+
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/headscale.svg
+      :align: center
+
+#########
+Headplane
+#########
+
+.. tag_list::
+
+Headplane_ is a feature-complete web UI for Headscale, allowing you to manage your nodes, networks, and ACLs with ease. It is open source and integrates directly with your Headscale instance.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Node.js <lang-nodejs>` and its package manager :manual_anchor:`npm <lang-nodejs.html#npm>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`web backends <web-backends>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+Headplane is released under the _[MIT License](https://github.com/tale/headplane/blob/main/LICENSE)_.
+
+All relevant legal information can be found here:
+
+  * https://github.com/tale/headplane/blob/main/LICENSE
+
+Prerequisites
+=============
+
+- Node.js 22 LTS or newer (see :manual:`Node.js <lang-nodejs>`)
+- PNPM 10.x (see https://pnpm.io/installation)
+- A running Headscale instance (see :lab:`Headscale <guide_headscale>`) on the same Uberspace account
```

Comment:
I'll stop the review here (for now) as this guide depends on #1904 which still needs work.